hack/create-operatorhub-prs.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+# Copyright 2023 Red Hat, Inc. and/or its affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+VERSION=$1
+if [[ $1 == v* ]]; then TAG=$1; else TAG=v$1; fi
+GITHUB_AUTHOR=$2
+COMMUNITY_OPERATORS=community-operators
+COMMUNITY_OPERATORS_PROD=community-operators-prod
+if [[ $3 == false ]]; then DRY_RUN=false; else DRY_RUN=true; fi
+if [ -z "$4" ]; then KOGITO_OPERATOR_DIR=$(pwd); else KOGITO_OPERATOR_DIR=$4; fi
+
+upstream_author=${UPSTREAM_AUTHOR:-kiegroup}
+
+if [ "$GITHUB_AUTHOR" = "$upstream_author" ]; then
+   echo "Upstream author and GitHub author are equal, no need to setup upstream ${upstream_author}"
+else
+    echo "Upstream author and GitHub author are equal. Adding upstream repo"
+    git remote add upstream https://github.com/${upstream_author}/kogito-operator.git >/dev/null 2>&1
+fi
+
+echo "Kogito Operator directory is ${KOGITO_OPERATOR_DIR}"
+
+git fetch --all --tags
+echo "Checking out Kogito $TAG"
+git checkout tags/$TAG
+
+TMP_WD=$(mktemp -d)
+echo "### Using the ${TMP_WD} tmp working dir ####"
+create_operatorhub_pr() {
+  cd $TMP_WD
+
+  echo "### Starting changes on $1 repo ####"
+  REPO_TO_CLONE="https://github.com/${GITHUB_AUTHOR}/$1"
+  echo "$1 directory does not exist, going to clone ${REPO_TO_CLONE}"
+  git clone ${REPO_TO_CLONE}
+
+  cd $1
+
+  git checkout -B kogito-$TAG
+  KOGITO_TAG_INNER_DIR=operators/kogito-operator/${VERSION}
+  mkdir -p ${KOGITO_TAG_INNER_DIR}
+  cp -rf ${KOGITO_OPERATOR_DIR}/bundle/app/* ${KOGITO_TAG_INNER_DIR}
+  cp -f  ${KOGITO_OPERATOR_DIR}/bundle.Dockerfile ${KOGITO_TAG_INNER_DIR}/Dockerfile
+  sed -i "s|bundle/app/manifests|manifests|g" ${KOGITO_TAG_INNER_DIR}/Dockerfile
+  sed -i "s|bundle/app/metadata|metadata|g" ${KOGITO_TAG_INNER_DIR}/Dockerfile
+  sed -i "s|bundle/app/tests|tests|g" ${KOGITO_TAG_INNER_DIR}/Dockerfile
+  git add --all
+  git commit --signoff -m "operator kogito-operator (${TAG})"
+  if [[ ${DRY_RUN} == false ]]; then
+    echo "We are running in non dry_run mode, going to push changes"
+    git push -u
```

Comment:
it is good to sign the pr too, and add a message.

Reply:
I didn't find anything related to the sign of a PR.

Since we are using the ```--fill``` option the PR will be filled with the info retrieved from the commit message.

We could put as a template into the body the ```docs/pull_request_template.md```

Reply:
But usually, it would be one of us that will be sending the PR, so the -s flag would take care of it, no?

Reply:
Are you suggesting to add the -s flag to the push command?

Because the commit is signed off using the --signoff, hh cli hasn't go a parameter to signoff the pull request.